core/src/main/java/org/elasticsearch/action/admin/cluster/allocation/ClusterAllocationExplanation.java

```diff
@@ -49,6 +54,9 @@
     private final Map<DiscoveryNode, Float> nodeWeights;
     private final UnassignedInfo unassignedInfo;
     private final long remainingDelayNanos;
+    private final List<IndicesShardStoresResponse.StoreStatus> shardStores;
+
+    private Map<DiscoveryNode, IndicesShardStoresResponse.StoreStatus> nodeStoreStatus = null;
```

Comment:
Do you mean the `transient` keyword? I'm not familiar with the `@Transient` annotation at all